modules/fluid_properties/include/fluidproperties/TabulatedFluidProperties.h

```diff
@@ -110,6 +112,23 @@ class TabulatedFluidProperties : public SinglePhaseFluidProperties
 
   virtual void vaporPressure(Real temperature, Real & psat, Real & dpsat_dT) const override;
 
+  template <typename T>
+  void vaporPressureTemplate(const T & temperature, T & psat, T & dpsat_dT) const;
```

Comment:
is this defined?

Reply:
Again, I was trying to keep it consistent with Water97 but I did forget to define it so I can just take it out.

Reply:
Sounds good. Let s remove that one

Reply:
removed.